inspec.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'r-train', '~> 0.10.4'
   spec.add_dependency 'thor', '~> 0.19'
-  spec.add_dependency 'json', '~> 1.8'
+  spec.add_dependency 'json', '>= 1.8', '< 3.0'
```

Comment:
json 2.0 requires ruby version 2.2, and a gem install doesn't take this into account (sadly), so relaxing this constraint will break gem installs on Ruby 2.1 and below.